src/app/Plans/components/VMMigrationDetails.tsx

```diff
@@ -105,7 +106,7 @@ const VMMigrationDetails: React.FunctionComponent = () => {
           migration.other.total / 1024
         )} GB`,
         {
-          title: <PipelineSummary total={0} current={0} status={migration.other.status} />,
+          title: <PipelineSummary status={migration.status2} />,
```

Comment:
@gildub Nice progress so far. Is this a typo? "status2"?

Reply:
That's because we didn't have clear visibility about the entire migration data. So in the meantime it's mocked.
Basically status2 is just a placeholder until that's resolved.